README.md

```diff
@@ -29,15 +29,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+> `map` returns values and will return a new array vs. `forEach` doesn't return anything, it calls a provided function on each element in the array
```

Comment:
Yes, correct, .map returns a new array. Or, more accurately, creates a new array. What it returns depends on how the function is setup.

forEach doesn't create a new array. Other than that point, they are pretty much the same. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach